### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by danyaljj**

### CODE REVIEW

**Comment by christos-c** [comment]
@danyaljj can you fix the conflicts?

**Comment by danyaljj** [comment]
Done.

File: lbjava/README.md [hunk 1]
```diff
@@ -18,8 +18,41 @@ Visit each link for its content
...
+
+## Compiling the code 
+To compile and package the LBJava code simply run:
+
+    mvn install
+
+NB: If you need to run `mvn clean` at any point, make sure to create `target/classes` directory before 
+running `mvn compile/install` since it is required for the java-source compilation process.
 
+## Using ILP inference
```
Comment (christos-c) [inline lbjava/README.md]: We have OJAlgo now too!

Comment (danyaljj) [inline lbjava/README.md]: Yeah I think I will update these later, when we are good place for the inference algorithms....

Comment (christos-c) [inline lbjava/README.md]: Fair enough

**Comment by danyaljj** [comment]
Hello

**Pull request merged**